defold-spine/src/comp_spine_model.cpp

```diff
@@ -800,6 +814,43 @@ namespace dmSpine
         dmRender::AddToRender(render_context, &ro);
     }
 
+    static void RenderListFrustumCulling(dmRender::RenderListVisibilityParams const &params)
+    {
+        DM_PROFILE("SpineModel");
+
+        SpineModelWorld* spine_world = (SpineModelWorld*)params.m_UserData;
+
+        const dmIntersection::Frustum frustum = *params.m_Frustum;
+        uint32_t num_entries = params.m_NumEntries;
+        for (uint32_t i = 0; i < num_entries; ++i)
+        {
+            dmRender::RenderListEntry* entry = &params.m_Entries[i];
+            int component_index = entry->m_UserData;
+
+            // get transformations for this entry
+            SpineModelComponent* component_p = spine_world->m_Components.Get(component_index);
+            dmTransform::Transform& trans = component_p->m_Transform;
```

Comment:
Unused.

Reply:
what is "Unused" ?

Reply:
> Unused.

What is "Unused" ?

Reply:
This `dmTransform::Transform& trans` is unused isn't it?